monai/visualize/img2tensorboard.py

```diff
@@ -126,3 +125,47 @@ def add_animated_gif_no_channels(writer, tag, image_tensor, max_out, scale_facto
                                                                     max_out=max_out, animation_axes=[1],
                                                                     image_axes=[2, 3], scale_factor=scale_factor),
                                           global_step)
+
+
+def plot_2d_or_3d_image(data, step, writer, max_channels=1, max_frames=64, tag='output'):
```

Comment:
need to clarify that `data` should have batch channel dimensions, this function only plot the first element in the batch (should we make it configurable?)

Reply:
Added doc-string in the latest patch.
User can config to plot which element in a batch now.
Thanks.